package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "test": "jest",
     "build": "webpack-cli",
-    "watch": "webpack-cli -w"
+    "watch": "webpack-cli -w",
+    "lint": "eslint . --ext .ts",
+    "lint-and-fix": "eslint . --ext .ts --fix"
```

Comment:
Semicolons are usually the way I see alternate commands built. i.e. `"lint:fix"`